e2e/support/helpers/e2e-visual-tests-helpers.js

```diff
@@ -190,7 +217,6 @@ function assertTooltipFooter({ name, value, secondaryValue }) {
 }
 
 export function assertEChartsTooltip({ header, rows, footer, blurAfter }) {
-  echartsTooltip().should("be.visible");
```

Comment:
`be.visible` does not work with fixed-position elements. simply calling echartsTooltip() already does the style assertions in isFixedPositionElementVisible plus the z-index and position assertions automatically.